gcp/apis/compute/v1alpha2/network_types.go

```diff
@@ -104,21 +105,22 @@ func (n *Network) SetReclaimPolicy(p v1alpha1.ReclaimPolicy) {
 
 // +kubebuilder:object:root=true
 
-// NetworkList contains a list of Network
+// NetworkList contains a list of Network.
 type NetworkList struct {
 	metav1.TypeMeta `json:",inline"`
 	metav1.ListMeta `json:"metadata,omitempty"`
 	Items           []Network `json:"items"`
 }
 
-// GCPNetworkSpec contains fields of googlecompute.Network object that are
-// configurable by the user, i.e. the ones that are not marked as [Output Only]
-// In the future, this can be generated automatically.
-type GCPNetworkSpec struct {
+// NetworkParameters define the desired state of a Google Compute Engine VPC
+// Network. Most fields map directly to a Network:
+// https://cloud.google.com/compute/docs/reference/rest/v1/networks
+type NetworkParameters struct {
```

Comment:
I was a little reluctant to rename this type, but I think doing so provides more consistency and predictability for API doc readers; the API doc handles embedded fields by linking to the embedded type and stating "Foo also supports all the fields of Bar", e.g. "NetworkSpec also supports all the fields of NetworkParameters". This change means every resource in stack-gcp has a `<T>Spec` field that embeds a `<T>Parameters` field.